### ISSUE TITLE
Pages 'flicker' when scrolling with touchpad

### ISSUE THREAD

**Issue by deccav** [elementary/applications-menu#34]
When scrolling between pages (I've only tried with 2 so far) using two fingers on a touchpad, the pages will occasionally flicker - as if I am scrolling between the two pages very quickly.

**Comment by cassidyjames** [comment]
@deccav thanks for the report! Is this still occurring for you?

**Comment by deccav** [comment]
Yes, it was still happening last time I tried about 2 weeks ago.

**Comment by danirabbit** [comment]
Since this was reported we switched to LibHandy Carousel, so this is probably fixed
